src/tests/contract-tests/consumer/sis-consumer.pact.test.ts

```diff
@@ -0,0 +1,81 @@
+import path from "path";
+import { PactV4, SpecificationVersion, MatchersV3 } from "@pact-foundation/pact";
+import type { UserIdentityResponseMetadata } from "../../../handlers/user-identity/user-identity-response-metadata";
+
+const { like } = MatchersV3;
+
+const USER_IDENTITY_ENDPOINT = "/user-identity";
+
+const provider = new PactV4({
+  consumer: "SisConsumerTests",
+  provider: "StoredIdentityService",
+  logLevel: "warn",
+  dir: path.resolve(process.cwd(), "pacts"),
+  spec: SpecificationVersion.SPECIFICATION_VERSION_V4,
+  host: "127.0.0.1",
+});
+
+describe("SIS Consumer", () => {
+  it("should return 200 if the stored identity exists", async () => {
+    const govukSigninJourneyId = "Test-" + Date.now();
+
+    await provider
+      .addInteraction()
+      .given("stored identity exists")
+      .uponReceiving("A userId for a stored identity that does not exist")
```

Comment:
Should this be "A userId for a stored identity that exists"?

I'm not sure I understand why it is "that does not exist" when the `given` is "stored identity exists". But I might be misunderstanding what this is doing.

Reply:
As discussed, resolved in the next PR, so will leave this.